antsibull/cli/antsibull_build.py

```diff
@@ -88,12 +88,16 @@ def _normalize_release_build_options(args: argparse.Namespace) -> None:
     if args.command not in ('single', 'multiple', 'rebuild-single'):
         return
 
+    if args.build_data_dir is None:
+        args.build_data_dir = args.dest_dir
```

Comment:
I'm okay with the antsibull-build script not having backwards compatibility right now, before ansible-2.10.0rc1.  I have a gut instinct that build_data_dir and dest_dir should be independent and independently default to the current working directory.  If backwards compatibility is not a concern, does that sound okay to you?

Reply:
I'm totally fine with that! Rebuilding of alpha versions will probably suffer, but well, these are alpha versions, so you need to get a fitting version of antsibull from the git history ;)